app/controllers/admin/reservations/item_pool_searches_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Admin
+  module Reservations
+    class ItemPoolSearchesController < BaseController
```

Comment:
This controller is mostly a clone of `Admin::ItemPoolsController`. If we add more/better searching we can do it in such a way that both views are improved.